cluster/aws/util.sh

```diff
@@ -237,8 +237,14 @@ function kube-up {
     ssh-keygen -f $AWS_SSH_KEY -N ''
   fi
 
-  aws iam get-instance-profile --instance-profile-name ${IAM_PROFILE} || {
-        echo "You need to set up an IAM profile and role for kubernetes"
+  detect-image
```

Comment:
It looks like git won the battle on this file - detect-image isn't defined in this file in this branch, and you have some merge conflicts below.